DataFormats/GeometryCommonDetAlgo/interface/GlobalErrorBase.h

```diff
@@ -1,7 +1,7 @@
 #ifndef GlobalErrorType_H
 #define GlobalErrorType_H
 
-#include "DataFormats/GeometryCommonDetAlgo/interface/DeepCopyPointer.h"
+#include "DataFormats/Common/interface/DeepCopyPointer.h"
```

Comment:
as commented in the main thread, I think that this include is superfluous (same in all other cases)